packages/tsup-configs/README.md

```diff
@@ -0,0 +1,73 @@
+# `@workleap/tsup-configs`
+Workleap's recommended [tsup](https://tsup.egoist.dev/) configs.
+
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](../../LICENSE)
+[![npm version](https://img.shields.io/npm/v/@workleap/tsup-configs)](https://www.npmjs.com/package/@workleap/tsup-configs)
+
+## Installation
+
+Install the package.
+
+**With pnpm**
+```shell
+pnpm add -D @workleap/tsup-configs
+```
+
+## Usage
+
+### Build a React or TypeScript Library Project with a watch mode
```

Comment:
I am not sure I would frame this around the "watch" mode.

It should be generelized strictly to the "dev" environment IMO because "watch" is only one feature that distinguish dev settings from build settings.

Reply:
It might be easier to explain and more concise if you merge the dev & build sections

Reply:
with the current setup, with "watch":true and "clean":false, the dev setup can only be used in watch mode. 

i guess we could do a `build` config, a `dev` config and a `watch` configs where:
- build: build for prod
- dev: build with sourcemaps / not minified / not treeshaken
- watch: used for the watchmode

-----------------------

I feel like the setup for projects that needs tsup with a dual dev/build setup is significatively different than a simple package that only has a build. Thats why i wanted to have 2 separate sections. I'll look at merging the two

Reply:
I've updated the texts.

Reply:
I am not sure what is the use case for a dev config without the watchmode?

Reply:
I agree. You mentionned that "watch" is only a small part of what a dev config is, but yet we think having a dev config without watch is useless. So i would argue that "watch" is a big part of the dev config.

Thats why i had it in a separate section

Reply:
Yeah but it's not only `watch`, there's also other things going on. We don't call the build config "minify" because a big part of it is to minify the source code.